src/positions.ts

```diff
@@ -57,7 +57,10 @@ export const findPositionsFromEvents = (options: DOMFunctions) => (
             map(event => ({ event, eventType: 'click' as 'click' })),
             // ignore click events caused by the user selecting text.
             // Selecting text should not mess with the hover, hover pinning nor the URL.
-            filter(() => window.getSelection().toString() === ''),
+            filter(() => {
+                const selection = window.getSelection()
+                return selection === null || selection.toString() === ''
+            }),
```

Comment:
@felixfbecker Does this look correct to you? `window.getSelection(): Selection | null` (it's nullable) in TS 3.5

Reply:
Perhaps `Boolean(window.getSelection())` works just as well, and is more concise.